openedx/core/djangoapps/programs/utils.py

```diff
@@ -0,0 +1,14 @@
+"""
+Helper methods for Programs.
+"""
+from edx_rest_api_client.client import EdxRestApiClient
+
+
+def programs_api_client(api_url, jwt_access_token):
+    """ Returns an Programs API client setup with authentication for the
+    specified user.
+    """
+    return EdxRestApiClient(
+        api_url,
+        jwt=jwt_access_token
```

Comment:
We want to pass our own generated JWT for this we have utility method `get_id_token` (same as we do for edxnotes).
This was also the reason we implemented `SuppliedJwtAuth` in the PR: https://github.com/edx/edx-rest-api-client/pull/12